src/index.ts

```diff
@@ -29,9 +38,13 @@ export default (
 ): Logger => {
   opts.redact = redact.addDefaultRedactPathStrings(opts.redact);
 
+  const trim = trimmer({
+    depth: (opts.maxObjectDepth ?? DEFAULT_MAX_OBJECT_DEPTH) - 1,
+  });
+
```

Comment:
Am torn between sharing a trimmer like this or just re-declaring for both default and custom serializers.